features/license_check.feature

```diff
@@ -0,0 +1,115 @@
+Feature: License check timer only runs in environments where necessary
+
+    @series.lts
+    @uses.config.machine_type.gcp.generic
```

Comment:
Since we are attaching with a contract token in the non-pro tests, I think we can make explicitly `@uses.config.contract_token` on those tests.